### DISCUSSION TITLE
Talk and collabora office is not working on nextcloud 25 after switching nginx serwer

### DISCUSSION THREAD

**Discussion by megatonix** [nextcloud/all-in-one#2435] [Questions] [answered]
Steps to reproduce
1. I use Nginx on differnet serwer eg. 192.168.0.10
2. Now i wanna change nginx to use serwer from 192.168.0.20
3. I add proxy host <domain_NC> and add new certs 
4.  After i switch on my router IP that works nginx from 192.168.0.10 to 192.168.0.20
5. Nextcloud works, but not Talk and Collabra

### Expected behavior
nextCloud works normally

### Actual behavior
I only switch different nginx IP. I can use my NextCloud app but i cannot send messenge to talk or open any file

### Host OS 
Ubuntu Server 22.04

#### Nextcloud AIO version 
25.0.5

#### Other valuable info 
![nginx](https://user-images.githubusercontent.com/56199368/235163659-66e2be24-8ba9-44c2-94be-9ed21920d1e0.png)

**Comment by szaimen** [comment]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Comment by megatonix** [comment, marked as answer]
I enabled websocket in nginx proxy manager it seems that helped. Sorry for bothering
